integreatly/integreatly-1.9.4/integreatly-operator.1.9.4.clusterserviceversion.yaml

```diff
@@ -0,0 +1,407 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: integreatlyoperator.1.9.4
+  namespace: placeholder
+  annotations:
+    alm-examples: "{}"
+    categories: Integreating greatly
+    certified: 'false'
+    containerImage: quay.io/integreatly/integreatly-operator:v1.9.4
+    createdAt: 2019-10-09 10:39:00
+    support: Integreatly
+    description: Greatly integrating integreatly greatness
+spec:
+  displayName: Integreatly
+  description: Integreatly installation and reconciliation operator
+  keywords: ['integreatly']
+  version: 1.9.4
+  maturity: alpha
+  replaces: integreatlyoperator.1.9.3
```

Comment:
Is this going to be an issue if we're skipping 1.9.3?

Reply:
yes, this needs to be changed to 1.9.2 @davidkirwan

Reply:
I think you're right, needs to be 1.9.2.